src/actions/MapActions.js

```diff
@@ -13,9 +14,13 @@ import {
 import {navKeys, tokens} from "../constants";
 
 
-export const initializeMap = (region) => {
+export const initializeMap = ({region}) => {
 
     return (dispatch) => {
+        dispatch({
+            type: MAP_IS_INITIALIZING
+        });
+
         firebase.firestore().collection('campsites')
```

Comment:
I realized that we accidentally had an infinite loop that was triggered when a new site was added (more on that below). The fix is to fire off a `MAP_IS_INITIALIZING` type that flips a flag in `AddSiteReducer` back to `false` for `sitesShouldUpdate`